testsuite/gnat2goto/tests/cfg_test_09/cfg_test_09.adb

```diff
@@ -0,0 +1,18 @@
+procedure CFG_Test_09 (X : in Integer; I : in out Integer) is
+begin
+   I := 0;
+   
+   if X > 0 then
+      loop
+	 exit when I - X > 20;
```

Comment:
Is Indentation correct here?

Reply:
It is not.  Will fix.

Reply:
I have used `gnatpp` to pretty print the code.  It looks better to me now.